os/mounting-storage.md

```diff
@@ -16,9 +16,13 @@ coreos:
         Type=ext3
 ```
 
-As you can see, it's pretty simple. You specify the attached device and where you want it mounted. Optionally, you can provide a file system type.
+[Cloud-Config][cloud-config] will store `media-ephemeral.mount` unit file into `/etc/systemd/system` directory and systemd will catch it on boot. As you can see, it's pretty simple. You specify the attached device and where you want it mounted. Optionally, you can provide a file system type.
 
-It's important to note that [systemd requires](http://www.freedesktop.org/software/systemd/man/systemd.mount.html) mount units to be named after the "mount point directories they control". In our example above, we want our device mounted at `/media/ephemeral` so it must be named `media-ephemeral.mount`.
+It's important to note that [systemd requires][systemd-mount] mount units to be named after the "mount point directories they control". In our example above, we want our device mounted at `/media/ephemeral` so it must be named `media-ephemeral.mount`.
```

Comment:
@joshix TODO: Sync with the lines from my previous PR describing mount unit naming convention.